src/FsLexYacc.Runtime/Lexing.fs

```diff
@@ -264,10 +264,10 @@ namespace FSharp.Text.Lexing
             LexBuffer<char>.FromChars (s.ToCharArray())
 
         static member FromTextReader (tr:System.IO.TextReader) : LexBuffer<char> = 
-           LexBuffer<char>.FromFunction(tr.Read) 
+           LexBuffer<char>.FromReadFunctions(Some tr.Read, Some (tr.ReadAsync >> Async.AwaitTask))
 
         static member FromBinaryReader (br:System.IO.BinaryReader) : LexBuffer<byte> = 
-           LexBuffer<byte>.FromFunction(br.Read) 
+           LexBuffer<byte>.FromFunction(br.Read)
```

Comment:
Why does that function even exist? The library was supposed to parse _text files_; not _binary_ ones.

At least, we could mark it as `Obsolete`.